### DISCUSSION TITLE
Question about stand alone variants

### DISCUSSION THREAD

**Discussion by SteJW** [duncanmcclean/simple-commerce#1059] [Feature Requests]
Hi,
We have products to sell with different variants, that do not lead to 1 specific product variant. We have a product configurator on the product detail page that basically does this:

- the product is a car model
- variant color red or blue
- variant transmission automatic or manual
- 2 doors or 4 doors
- airconditioning yes or no

Each variant has a different price and we dont really need to create combination prices. This is an example, but it reflects our product.

So the question is:
- how can we show multiple variants on the product page e.g. 4 select fields with their options?
- how can we add multiple variants and the product to the cart. It looks like a single variant is supported, but i think we're looking for an array option?

If it's not possible, can you point us in the right direction on how to change/extend this?

**Comment by duncanmcclean** [comment]
I don't think the included Products Variants feature will do what you're after. 

Instead, what you could do is let the user configure those options and pass them through as "metadata fields" when you add the product to the cart. [see docs](https://simple-commerce.duncanmcclean.com/tags/cart#content-add-item-to-cart)

Then, based on the configuration options the user chose, you could use Simple Commerce's [Dynamic Pricing](https://simple-commerce.duncanmcclean.com/dynamic-pricing) feature to  dynamically "generate" the price for the product.

**Comment by SteJW** [comment]
Thanks for your quick reply Duncan! We will have a look.
